packages/app/src/app/page.tsx

```diff
@@ -1,17 +1,20 @@
+'use client'
```

Comment:
You can replace 'use-client' to the 'styled.ts' file

Reply:
tried to replace, but it's not worked